release-notes.md

```diff
@@ -1,5 +1,70 @@
 # Release Notes
 
+## 3.0
+
+medic-conf v3.0 contains breaking changes! This release only impacts the `compile-app-settings` action and the partner code written in `tasks.js`, `targets.js`, `contact-summary.js`, and `contact-summary.templated.js`.
+
+### Breaking Change - Static Analysis via EsLint
+medic-conf has changed the engine we use for performing static analysis from [JsHint](https://jshint.com/) to [eslint](https://eslint.org/). This grants authors more control over the static analysis rules which are enforced on their code. We provide a set of default rules, but you can create an [.eslintrc file](https://eslint.org/docs/user-guide/configuring#using-configuration-files-1) to override these defaults or add rules. The default static analysis rules which are enforced in medic-conf 3.0 are similar to the rules which were enforced in medic-conf 2.x, but details may be slightly different.
+
+### Support for ECMAScript 6
+In previous versions of medic-conf, Javascript was limited to ES5. [ECMAScript 6](http://es6-features.org/) was a huge change for the JavaScript language and community. To use ES6 in your JavaScript configuration code, add a `.eslintrc` file to your project directory with the following contents:
+
+```
+{
+  "env": {
+    "node": true,
+    "es6": true
+  },
+  "parserOptions": {
+    "ecmaVersion": 6
+  }
+}
+```
+
+If your project is using [medic-android](https://github.com/medic/medic-android) you must be running >v0.1.203 (March 2017) to use `ecmaVersion: 6`. The `medic-android` project does not support ecmaVersions above 6.
```

Comment:
The use of project in two different ways here is a little confusing. Could replace the second as "`medic-android` app". Also, may help to flesh out ">v0.1.203" to "greater than version 0.1.203" (or is it intended to be "version 0.1.203 or greater"?). Either way, would help clarify. The date is a nice touch!